### DISCUSSION TITLE
Native Unix software in Airyx ?

### DISCUSSION THREAD

**Discussion by m77it** [ravynsoft/ravynos#230] [Q&A]
Dear all,
I have a couple of old Unix software that cannot run under Linux. Airyx is a possible solution? Suggestions?
Thanks
Marco

**Comment by mszoek** [comment]
> I have a couple of old Unix software that cannot run under Linux. Airyx is a possible solution? Suggestions?

Is this software you have source code for or binaries? What Unix are they for?

**Reply by m77it** [reply]
Not sure about the Unix version. It is dated 1988. I have the code, it if Fortran

**Reply by mszoek** [reply]
It *might* work - BSD is closer to Unix™ than Linux is, but I suspect the odds are not great. There is nothing special about airyxOS in this regard; in fact, the changes we've made might hurt rather than help. Plain FreeBSD or the open Solaris (OpenIndiana) are probably the best bets at building & running something like that.

**Reply by m77it** [reply]
Thanks. I will try Solaris of freeBSD
Regards,
Marco
